contexts/did-v0.11.jsonld

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
The contexts live in https://github.com/w3c/did-spec-registries/tree/master/contexts, not here

Reply:
Mmm, not sure why the .json-ld files got included.